src/main/resources/META-INF/services/com.mycila.maven.plugin.license.PropertiesProvider

```diff
@@ -0,0 +1 @@
+org.hawkular.build.license.CopyrightRangeProvider
```

Comment:
Is the new line really necessary? I never used this kind of service files before. It works also without the new line.